enumerable_method.rb

```diff
@@ -0,0 +1,177 @@
+# frozen_string_literal: true
+
+module Enumerable #:nodoc:
+  # my_each method
+
+  def my_each
+    return to_enum unless block_given?
+
+    i = 0
+    while i < size
+      yield(to_a[i])
+      i += 1
+    end
+    self
+  end
+
+  # each_with_index method
```

Comment:
remove this comment and any you have in this file.

Reply:
> remove this comment and any you have in this file.

Fixed!